src/model/likes.model.ts

```diff
@@ -86,22 +86,22 @@ export default {
     const idLike = await prisma.likes.findFirst({
       where: {
         users_id: userId,
-        routines_id: routineId
+        routines_id: routineId,
       },
       select: {
         id: true,
-      }
+      },
     });
     return idLike;
   },
 
-  async deleteLike(id: number){
+  async deleteLike(id: number) {
     const deletLike = await prisma.likes.delete({
```

Comment:
Another minor spelling suggestion: deletLike => deleteLike

Reply:
Oh i see now, it would have the same name as the function. Still, maybe there is a better name.

Reply:
Like I said in previous comment we can create an issue where we can check for spelling errors and also better naming! Good catch